components/visuals/TravelsByDepartmentPrismMap.tsx

```diff
@@ -0,0 +1,112 @@
+import { scaleOrdinal, scaleLinear } from "d3";
+import { GeoProjection } from "d3-geo";
+import { GeoJsonProperties } from "geojson";
+import { FC, useState } from "react";
+import { ExtrudeGeometryOptions } from "three";
+import type { Topology } from "topojson-specification";
+import PrismMap from "../PrismMap";
+import clsx from "clsx";
+import {
+  AccumulativeShadows,
+  Environment,
+  OrbitControls,
+  RandomizedLight,
+} from "@react-three/drei";
+import { Canvas } from "@react-three/fiber";
+import { BtorsGroupedByCountryByDepartment } from "../../lib/data/queries/btors/getBtorsGroupedByCountryByDepartment";
+
+type Props = {
+  btorsByCountryByDepartment: BtorsGroupedByCountryByDepartment;
+  topology: Topology;
+  topologyObject: string;
+  projection: GeoProjection;
+  width: number;
+  length: number;
+  extrudeGeometryOptions?: ExtrudeGeometryOptions;
+};
+
+const TravelsByDepartmentPrismMap: FC<Props> = ({
+  btorsByCountryByDepartment,
+  topology,
+  topologyObject,
+  projection,
+  width,
+  length,
+  extrudeGeometryOptions = {},
+}) => {
+  const ITC = ["EOS", "AES", "GIP", "NRS", "PGM", "WRS", "VAR"];
+  const [activeDepartment, setActiveDepartment] = useState<string | undefined>(
+    undefined,
+  );
+  const map = new Map<string, GeoJsonProperties>();
+  Array.from(btorsByCountryByDepartment)
+    .map((d) => {
+      d[1]
+        .filter((d1) => d1.departmentId == activeDepartment)
+        .map((d2) =>
+          d2
+            ? map.set(d2.country, {
+                value: d2.count,
+                category: "important",
+              })
+            : [],
+        );
```

Comment:
I think variable names like, in this case, `department` or `country` could increase the legibility of the code (instead of names like `d1`)

Reply:
Ok！